examples/machinelearningservices/computeinstance.yaml

```
@@ -0,0 +1,187 @@
+apiVersion: machinelearningservices.azure.upbound.io/v1beta1
+kind: ComputeInstance
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/computeinstance
+    uptest.upbound.io/timeout: "2400" # 40 min timeout
+  labels:
+    testing.upbound.io/example-name: exampleci0001
+  name: exampleci0001
+spec:
+  forProvider:
+    authorizationType: personal
+    description: foo
+    location: West Europe
+    machineLearningWorkspaceIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: exampleworkspace001
+    subnetResourceIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    tags:
+      foo: bar
+    virtualMachineSize: STANDARD_DS11_V2
+---
+apiVersion: insights.azure.upbound.io/v1beta1
+kind: ApplicationInsights
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/computeinstance
+  labels:
+    testing.upbound.io/example-name: exampleinsights
+  name: exampleinsights
+spec:
+  forProvider:
+    applicationType: web
+    location: West Europe
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgcomputeinstance01
+---
+apiVersion: alertsmanagement.azure.upbound.io/v1beta1
+kind: MonitorSmartDetectorAlertRule
+metadata:
+  annotations:
+    meta.upbound.io/example-id: alertsmanagement/v1beta1/monitorsmartdetectoralertrule
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    name: myrule
+    actionGroup:
+      - idsRefs:
+          - name: example
+    detectorType: FailureAnomaliesDetector
+    frequency: PT1M
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgcomputeinstance01
+    scopeResourceIdsRefs:
+      - name: exampleinsights
+    severity: Sev3
+---
+apiVersion: insights.azure.upbound.io/v1beta1
+kind: MonitorActionGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: alertsmanagement/v1beta1/monitorsmartdetectoralertrule
```

```suggestion
    meta.upbound.io/example-id: machinelearningservices/v1beta1/computeinstance
```

Could you please fix the example-id?

---

Done